docs/sources/writing-guide/front-matter/index.md

```diff
@@ -47,6 +47,16 @@ The following headings describe what each element does and provides guidelines f
 Provides an HTML redirect from the pages in the list to the current page.
 For more information, refer to [Hugo aliases](#hugo-aliases).
 
+### date
+
+Describes the initial publish date of the page.
+Hugo produces XML page outputs for use by RSS feeds where users can be notified of updates.
+Customers use RSS feeds of our release notes pages to be notified of new releases.
+Therefore, the `date` front matter is recommended for release note pages.
```

Comment:
does this apply to what's new (we don't have Grafana release notes)

Reply:
I think we should start to put publish dates in our What's New pages, yes.